riscv-executor/src/lib.rs

```diff
@@ -1039,10 +1043,22 @@ impl<'a, 'b, F: FieldElement> Executor<'a, 'b, F> {
 
         self.proc.backup_reg_mem();
 
-        set_col!(X, get_col!(X_const));
-        set_col!(Y, get_col!(Y_const));
-        set_col!(Z, get_col!(Z_const));
-        set_col!(W, get_col!(W_const));
+        if self.proc.col_is_defined("X_const") {
```

Comment:
Column name needs the namespace here `main::X_const`.
Still this is not quite right: `X_const` is being set from the "rom" (`p_X_const`), which is then being used to set `X` here.

Reply:
But for the purposes of this PR just fixing the column names here is fine (witness is not being checked in tests for now)